manifests/params.pp

```diff
@@ -29,9 +29,21 @@
   if $::operatingsystem == 'Ubuntu' {
     if versioncmp($::lsbdistrelease, '8.04') < 1 {
       $init_style = 'debian'
-    } else {
+    } elsif versioncmp($::operatingsystemrelease, '15.04') < 0 {
```

Comment:
Can we either use `$::lsbdistrelease` here or replace the above with `$::operatingsystemrelease` for consistency sake? I'm in favor of the latter because `$::operatingsystemrelease` is used everywhere else.